src/components/DroughtThresholds.vue

```diff
@@ -829,144 +812,115 @@ export default {
 $writeFont: 'Nanum Pen Script', cursive;
 $base: 0.6rem; //for chevron scroll animation
 // frames are stacked and class is added on an off w/ scroll trigger to bring to front
-
-
-#title-container {
-  position: fixed;
-  background-size: cover;
-}
-.knockout-text {
-  font-size: 2.5em;
-  font-weight: bold;
-  font-family: Roboto, 'Helvetica Neue', Arial, sans-serif;
-  color: #032a56;
-  padding: 0 0 0 50px;
-  @media (min-width: 900px){
-    font-size:3.0em;
-  }
-}
-.bigger {
-  font-size: 15vh;
-}
+$usgsBlue: #032a56;
 
 .grid-container{
   display: grid;
-  grid-template-columns: 1fr;
-  grid-template-rows: 0.5fr;
+  grid-template-columns: 3fr 0.5fr;
+  grid-template-rows: auto 10em auto;
   grid-template-areas:
-    "textbox"
-    "chart";
-  grid-gap: 0;
-  align-content: start;
-  width:100%;
-  height: 80vh;
+    "title chevron"
+    "textbox textbox"
+    "chart chart";
+  justify-content: center;
+  margin: auto;
+  width:95vw;
   position: sticky;
   left: 10px;
-  top: 35vh;
-  @media (min-width: 900px){
-    grid-template-columns: 1fr 3fr;
+  top: 81px;
+  height: 88vh;
+  @media (min-width: 950px){
+    width: 70vw;
+    max-width: 1400px;
+    grid-template-columns: minmax(100px, 400px) auto 1fr;
+    grid-template-rows: 0.5fr 3fr;
     grid-template-areas:
-      "textbox chart"
+      "title title chevron"
+      "textbox chart chart"
+  }
+}
+.title-text {
+  grid-area: title;
+  align-self: center;
+  font-size: 2em;
+  padding: 40vh 0 40vh 0;
+  color: $usgsBlue;
+  @media (min-width: 950px){
+    font-size:3.0em;
+  }
+  transition: ease 1s;
+  &--scrolled {
+    padding: 5px 0 5px 0;
   }
 }
-
 #hydro-chart-container{
   font-size: 1.2rem;
   grid-area: chart; // names the grid child component
-  place-self: center;
+  align-self: center;
+  justify-self: center;
+  height: 95%;
+  width: 95%;
+  max-height: 60vh;
   display: flex;
     display: -webkit-flex;
     justify-content: space-between;
     -webkit-justify-content: space-between;
-}
-
-
-
-#textbox-container{
-  grid-area: textbox; // names the grid child component
-}
-.text-container {
-  border-radius: 25px;
-  background: #FFE3AD;
-  position:absolute;
-  p{
-    padding: 15px;
-  }
-  @media (min-width:900px) {
-    max-width:25%;
+  @media (min-width: 950px){
+    height: 60vh;
   }
 }
-
-#scroll-container {
-  padding: 5rem 1rem;
+.textBox {
+  grid-area: textbox;
+  align-self: center;
+  justify-self: center;
+  color: $usgsBlue;
```

Comment:
Nice, this will be good if we also end up using this blue as the streamflow line on the data graph, too.